pandas/core/ops.py

```diff
@@ -265,6 +267,18 @@ def _get_frame_op_default_axis(name):
 -------
 result : Series
 
+Examples
+--------
+>>> a = pd.Series([1, 1, 1, np.nan], index=['a', 'b', 'c', 'd'])
```

Comment:
show a and b as well

Reply:
You need to have a line with just `>>> a` on it before showing the Series. Likewise for `b`.